packages/components/src/components/positions-drawer-card/positions-drawer-card.jsx

```diff
@@ -118,8 +121,8 @@ const PositionsDrawerCard = ({
     const supported_contract_card = (
         <div
             className={classNames('dc-contract-card', {
-                'dc-contract-card--green': !is_multiplier && profit_loss > 0 && !result,
-                'dc-contract-card--red': !is_multiplier && profit_loss < 0 && !result,
+                'dc-contract-card--green': !is_accumulator && !is_multiplier && profit_loss > 0 && !result,
+                'dc-contract-card--red': !is_accumulator && !is_multiplier && profit_loss < 0 && !result,
```

Comment:
Seems like we use the same profit and loss condition 3 times in this file. Can we move it to a constant declaration and reuse?